Jenkinsfile

```diff
@@ -59,6 +59,19 @@ pipeline {
             }
             }
         }
+        stage("approx coverage"){
+        //coverage report
+        agent{  label utils.rocmnode("tunatest") }
+        steps {
+            script{
+                try {
```

Comment:
@reidkwja  please remove this try-catch and let it fail. Work with @msaudulhassan  to fix the failing SolverAnalytics test.

Reply:
Resolved